### PR TITLE
updated header pacni

### PR DESCRIPTION
**Pull request by KMDOFNI**
updated header pacni

### CODE REVIEW

File: project/sites/pacni/themes/pacni_theme/src/scss/components.scss [hunk 1]
```diff
@@ -7,6 +7,7 @@
 
 #header {
   background: resolve('header_bg.jpg') no-repeat top right;
+  margin: 5px auto 0.1rem;
```
Comment (soda736) [inline project/sites/pacni/themes/pacni_theme/src/scss/components.scss:10]: Should keep the unit consistent and you don't need the 0 in a decimal.
`margin: .5rem auto .1rem`

**Review by soda736 — CHANGES_REQUESTED** [review]

File: project/sites/pacni/themes/pacni_theme/src/scss/components.scss (hunk 1 above)
Comment (KMDOFNI) [reply project/sites/pacni/themes/pacni_theme/src/scss/components.scss]: Thanks , this has now been updated

File: project/sites/pacni/themes/pacni_theme/src/scss/components.scss [hunk 2]
```diff
@@ -7,6 +7,7 @@
 
 #header {
   background: resolve('header_bg.jpg') no-repeat top right;
+  margin: 5rem auto .1rem;
```
Comment (soda736) [inline project/sites/pacni/themes/pacni_theme/src/scss/components.scss:10]: I think this was supposed to be .5rem and not 5rem?

**Review by soda736 — CHANGES_REQUESTED** [review]

File: project/sites/pacni/themes/pacni_theme/src/scss/components.scss (hunk 2 above)
Comment (KMDOFNI) [reply project/sites/pacni/themes/pacni_theme/src/scss/components.scss]: Thanks this has been updated now

**Review by soda736 — APPROVED** [review]

**Pull request merged**